src/math/ColorManagement.js

```diff
@@ -12,16 +12,31 @@ export function LinearToSRGB( c ) {
 
 }
 
-// JavaScript RGB-to-RGB transforms, defined as
-// FN[InputColorSpace][OutputColorSpace] callback functions.
+// RGB-to-RGB transforms, defined as `FN[InputColorSpace][OutputColorSpace] → conversionFn`.
 const FN = {
 	[ SRGBColorSpace ]: { [ LinearSRGBColorSpace ]: SRGBToLinear },
 	[ LinearSRGBColorSpace ]: { [ SRGBColorSpace ]: LinearToSRGB },
 };
 
 export const ColorManagement = {
 
-	legacyMode: true,
+	enabled: false,
+
+	get legacyMode() {
+
+		console.warn( 'THREE.ColorManagement: .legacyMode=false renamed to .enabled=true.' );
```

Comment:
Please add the release when the property has been renamed so we can easily removed the code after ten releases.
```js
console.warn( 'THREE.ColorManagement: .legacyMode=false has been renamed to .enabled=true. in r147' );
```

Reply:
Example:

https://github.com/mrdoob/three.js/blob/735d376fc58a3b78d58fdb8dacd47bd6fd4b2d69/src/core/BufferAttribute.js#L346-L352